pkg/cloudprovider/subnets_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	. "github.com/aws/karpenter-core/pkg/test/expectations"
 )
 
-var _ = Describe("Subnets", func() {
```

Comment:
Can we make sure to move some of these tests that still apply to the cloudprovider to the `suite_test` in the cloudprovider package?

Reply:
Moved the remaining tests